ethereum/permissioning/src/test/java/org/hyperledger/besu/ethereum/permissioning/account/AccountPermissioningControllerFactoryTest.java

```diff
@@ -92,14 +92,14 @@ public void createLocalConfigOnlyControllerShouldReturnExpectedController() {
   }
 
   @Test
-  public void createOnchainConfigWithAccountPermissioningDisabledShouldReturnEmpty() {
-    SmartContractPermissioningConfiguration onchainConfig =
+  public void createFlexibleConfigWithAccountPermissioningDisabledShouldReturnEmpty() {
```

Comment:
this class is permissioning so let's keep "Onchain" here

Reply:
ee64864